packages/forms/src/elements/MultiThumbRange.spec.tsx

```diff
@@ -57,11 +67,103 @@ describe('MultiThumbRange', () => {
     });
   });
 
+  describe('MultiThumbRange label', () => {
+    test('clicking the label focuses the min thumb', () => {
+      const { getByText, getAllByRole } = render(
+        <Field>
+          <Label>MultiThumbRange</Label>
+          <MultiThumbRange />
+        </Field>
+      );
+      const label = getByText('MultiThumbRange');
+      const minThumb = getAllByRole('slider')[0];
+
+      expect(minThumb).not.toHaveFocus();
+      userEvent.click(label);
+      expect(minThumb).toHaveFocus();
+    });
+
+    test('focus leaves the min thumb when user blurs out of min thumb', () => {
+      const { getByText, getAllByRole } = render(
+        <Field>
+          <Label>MultiThumbRange</Label>
+          <MultiThumbRange />
+        </Field>
+      );
+      const label = getByText('MultiThumbRange');
+      const minThumb = getAllByRole('slider')[0];
+
+      expect(minThumb).not.toHaveFocus();
+      userEvent.click(label);
+      expect(minThumb).toHaveFocus();
+
+      userEvent.tab();
+      expect(minThumb).not.toHaveFocus();
+    });
+
+    test('hovering over the label visually identifies the min thumb', () => {
+      const { getByText, getAllByRole } = render(
+        <Field>
+          <Label>MultiThumbRange</Label>
+          <MultiThumbRange />
+        </Field>
+      );
+      const label = getByText('MultiThumbRange');
+      const minThumb = getAllByRole('slider')[0];
+
+      expect(minThumb).toHaveStyleRule('border-color', getColor('blue', 600));
+      expect(minThumb).toHaveStyleRule('background-color', getColor('blue', 600));
```

Comment:
We apply `data-garden-*` testing values in the DOM. Could we reference these instead of comparing CSS values?

Reply:
Wouldn't asserting against `data-garden-*` make this a test brittle which encourages assertions against implementation details over user experience?

What if a developer accidentally mistyped the CSS property? Or maybe they provided an incorrect color? Or the CSS properties were accidentally removed? The `data-garden-*` attributes would still be valid, the test still passes ✅ , but the user would see a broken UI.